tests/conftest.py

```diff
@@ -41,3 +41,13 @@ def cube(cubes):
     from CSET.operators import constraints, filters
 
     return filters.filter_cubes(cubes, constraints.generate_cell_methods_constraint([]))
+
+
+@pytest.fixture(scope="session")
+def vertical_profile_cube():
+    """Get a vertical profile Cube."""
+    from CSET.operators import read
+
+    return read.read_cube(
+        "tests/test_data/air_temperature_vertical_profile_as_series.nc"
+    )
```

Comment:
```suggestion
@pytest.fixture(scope="session")
def vertical_profile_cube_readonly():
    """Get a vertical profile Cube. It is NOT safe to modify."""
    from CSET.operators import read

    return read.read_cube(
        "tests/test_data/air_temperature_vertical_profile_as_series.nc"
    )


@pytest.fixture()
def vertical_profile_cube(vertical_profile_cube_readonly):
    """Get a vertical profile Cube.  It is safe to modify."""
    return vertical_profile_cube_readonly.copy()
```
Echoing the changes on #631, this means we don't need to be as careful with using these cubes.